app/utils/store.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, bucket=None):
     def init_app(self, app):
         self.bucket = app.config['DOCUMENTS_BUCKET']
 
-    def put(self, service_id, document_stream, *, mimetype='application/pdf'):
+    def put(self, service_id, document_stream, *, sending_method, mimetype='application/pdf'):
```

Comment:
I know it's not you, wondering why we have a `*` in the middle? Can we get rid of it?

Reply:
sure